clients/python-wrapper/pylotl/client.py

```diff
@@ -0,0 +1,114 @@
+import base64
+import binascii
+import copy
+from typing import Optional
+import httpx
+
+import lakefs_sdk
+from lakefs_sdk.client import LakeFSClient
+
+from pylotl.config import ClientConfig
```

Comment:
PLEASE let's not call this pylotl :)
I think the package name should simply be lakeFS.